Makefile

```diff
@@ -64,11 +71,18 @@ psp_rr-objs := psp_rr_main.o psp_rr.o rr.o psp_lib.o $(lib)
 
 ext-libs := -lm -lrt -lpthread
 
+tcpdirect.o: tcpdirect.cu
+	nvcc -arch=sm_90 -O3 -g -D_GNU_SOURCE -DNO_LIBNUMA -DWITH_TCPDIRECT -c -o $@ $^
```

Comment:
I don't know if it's possible to retain the CFLAGS here, but that would be nice, if possible, but no big deal.